nibe/connection/nibegw.py

```diff
@@ -153,6 +158,62 @@ def datagram_received(self, data, addr):
             elif cmd == "PRODUCT_INFO_MSG":
                 with suppress(InvalidStateError, CancelledError, KeyError):
                     self._futures["product_info"].set_result(msg.fields.value.data)
+            elif cmd == "RMU_DATA_MSG":
+                data = msg.fields.value.data
+                self._on_coil_value(40004, (data.bt1_outdoor_temperature - 5) / 10)
+                self._on_coil_value(40013, (data.bt7_hw_top - 5) / 10)
+
+                def on_set_point(
+                    use_room_sensor: bool,
+                    value: int,
+                    room_address: int,
+                    offset_address: int,
+                ):
+                    if use_room_sensor:
+                        self._on_coil_value(room_address, (value + 50) / 10)
```

Comment:
This +50 here make sense since it is sent as a 8bit value, with scale of 10. So max without the offset would be 25.5 degrees. Now the max is 30.5.